docs/rules/vue-exports-component-directive.md

```diff
@@ -0,0 +1,35 @@
+# vue-exports-component-directive
```

Comment:
The tests fail because you need to rebuild this file (so that the other test cases are added)

Reply:
I didn't realize the docs were supposed to list all of the test cases - doing

Reply:
Except we can't just dump the expected output because its wrong - looking at the test failure at https://github.com/wikimedia/eslint-plugin-mediawiki/pull/65/checks?check_run_id=3174536594
* there shouldn't be a `;` after the closing `</script>` tag
* syntax highlighting should be for "vue" not "js"
* its unclear that the example of a non-vue file is indeed a non vue file

Reply:
Ugh, wow, that's super broken. It looks like this bug also affects https://github.com/wikimedia/eslint-plugin-mediawiki/blob/master/docs/rules/no-vue-dynamic-i18n.md , though it's subtler there (semicolons after `</template>`)

Reply:
yeah, I noticed that - I'm going to send a patch to fix that too (discovered that the documentation generating code is also a wikimedia package), but for now I'll deal with it by having it be a bit off